invenio_app_ils/ui/main/src/index.js

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
+// import 'react-app-polyfill/ie9'; // For IE 9-11 support
```

Comment:
I consider it a confusing practice to commit commented code soon we will have no clue why is there and if works or not.

Reply:
Yes leftover!I will un comment it basically cause it's needed for IE 9-11 support.